corehq/apps/app_manager/templates/app_manager/form_view_base.html

```diff
@@ -103,9 +103,18 @@
             var isModuleFilterEnabled = {{ is_module_filter_enabled|BOOL }};
             var formFilter = {{ form.form_filter|JSON }};
             var allOtherFormsRequireACase = {{ form.all_other_forms_require_a_case|BOOL }};
+            var formFilterReferencesCase = {{ form_filter_references_case|BOOL }};
             var formFilterAllowed = ko.computed(function () {
```

Comment:
On Wed, Mar 30, 2016 at 4:15 PM, Amelia <notifications@github.com> wrote:

> In corehq/apps/app_manager/templates/app_manager/form_view_base.html
> <https://github.com/dimagi/commcare-hq/pull/10972#discussion_r57955725>:
>
> > @@ -103,9 +103,18 @@
> >              var isModuleFilterEnabled = {{ is_module_filter_enabled|BOOL }};
> >              var formFilter = {{ form.form_filter|JSON }};
> >              var allOtherFormsRequireACase = {{ form.all_other_forms_require_a_case|BOOL }};
> > +            var formFilterReferencesCase = {{ form_filter_references_case|BOOL }};
> >              var formFilterAllowed = ko.computed(function () {
>
> Hey,
> Is this changing the behavior of the UI to allow you to enter form filter
> conditions when previously you weren't? And potential bad configurations we
> disallowed before are now allowed, but caught on build?
>
yes, that's how it will behave.  it will also show an error message in the
form filter ui once you refresh.

> It's possible that this should be lumped into the module display condition
> feature preview instead of allowing more configurations productized.
>
not sure, i think it's getting confusing using that feature preview to
restrict/allow certain form filter conditions

> Longer term, the way I see this productized is always allowing form filter
> conditions but using special syntaxing to only autocomplete to references
> that are available
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/dimagi/commcare-hq/pull/10972/files/bc5d1610ffd801eb752edd42c802def1ef092786#r57955725>
>